.idea/compiler.xml

```diff
@@ -27,10 +27,6 @@
       <module name="google-account-plugin_test" target="1.7" />
       <module name="google-cloud-tools-plugin_main" target="1.7" />
       <module name="google-cloud-tools-plugin_test" target="1.7" />
-      <module name="jps-plugin_main" target="1.7" />
```

Comment:
After the source context change, we'll need to leave this bit here I think (even though I'm not too sure what this does).